lib/Runtime/Library/JavascriptFunction.cpp

```diff
@@ -2302,14 +2302,14 @@ void __cdecl _alloca_probe_16()
         {
             return false;
         }
-        Js::FunctionBody* funcBody = func->GetFunctionBody();
-        bool isWAsmJs = funcBody->GetIsAsmJsFunction();
-        bool isWasmOnly = funcBody->IsWasmFunction();
+
+        bool isWAsmJs = AsmJsScriptFunction::Is(func);
```

Comment:
>isWAsmJs [](start = 13, length = 8)

Should this be isAsmJS instead of isWAsmJs?

Reply:
WasmScriptFunction are still AsmJsScriptFunction, therefore, this Boolean still means that we have either an asmjs function or wasm function

Reply:
yes but better to rename just to isAsmJs as below bool takes care of wasm;